test/__tests__/DefaultFormGenerator.test.js

```diff
@@ -11,13 +11,13 @@ describe('Default form generator', () => {
 
   it('generates empty one-step wizard as a default form', async () => {
     const form = DefaultFormGenerator.generateForm();
-    const [formProperties, structure] = await FormGenerator.constructDefaultForm(null);
+    const [formQuestions, structure] = FormGenerator.constructDefaultForm(null);
 
-    expect(formProperties.formQuestions.length).toEqual(1);
+    expect(formQuestions.length).toEqual(1);
 
     expect(structure).toEqual({ root: form['@graph'][0] });
```

Comment:
Now I am confused. Didn't you just remove "root: " ... in DefaultFormGenerator.js ... does it really work?

Reply:
@blcham Yes it actually works now, I also tested on actual stories to check. Here is what I was getting before:
![image](https://user-images.githubusercontent.com/57750426/162141281-a3e2d39e-761a-4b65-8cc5-056a20811140.png)

I should also be changed for other forms in form generator

Reply:
ok thanks